lib/cdo/pegasus/donor.rb

```diff
@@ -0,0 +1,14 @@
+# Returns a random donor's twitter handle.
```

Comment:
ideally this would be in shared and we would specify access to the pegasus DB so that dashboard can use this function and we don't have duplicated code.

ideally this one would actually DB data instead of being faked in JS: https://github.com/code-dot-org/code-dot-org/blob/1900428aa041ac2a44fc465f4aff7b7dcbc7161d/apps/src/util/twitterHelper.js#L1

Reply:
I think there's at least one more place where we make this function but I'm still looking for it.

Reply:
https://github.com/code-dot-org/code-dot-org/blob/256c411ce95ca2514e44b9924cfbfd57ef2cdc15/lib/cdo/user_helpers.rb#L63

Reply:
https://github.com/code-dot-org/code-dot-org/blob/60e7bd2fbcedfd9bfc845047f34cb5a7d0a0a433/lib/cdo/graphics/certificate_image.rb#L105

Reply:
^Plus none of these ones have all the fall back safety of the function here, and don't seem to be tested at all. 

Overall it'd be great to test this one and remove the others, but it's non-blocking for hoc and this PR. Great werketing item to track though.

Reply:
> ideally this would be in shared and we would specify access to the pegasus DB so that dashboard can use this function and we don't have duplicated code

What do you mean by this?  By moving the code to `/lib/cdo` we are re-using it in Dashboard.

Reply:
oh I didn't realize this, cool